draft-ietf-acme-acme.md

```diff
@@ -1801,6 +1802,39 @@ If all of the above verifications succeed, then the validation is successful.
 If no DNS record is found, or DNS record and response payload do not pass these
 checks, then the validation fails.
 
+# Use with CAA Records
+
+An ACME server SHOULD support CAA DNS records as described in {{RFC6844}}. The
+server SHOULD look for such records when issuing authorizations, as opposed to
+when issuing certificates.
```

Comment:
I do wonder about the use case for long authorizations. Since the whole purpose of ACME is to automate issuance, an authorization need only be long enough for an ACME client to conclude the immediate issuance transactions it needs to make...?